docs/documentation.rst

```diff
@@ -0,0 +1,8 @@
+Explain PRIMO in One Page
+=========================
+
+A documentation has been created to demonstrate PRIMO in non-technical terms. 
+
```

Comment:
I don't think we need a header here. If the title of the page is "PRIMO explained in one page" or something like that, then I think that the header is not needed.

Reply:
Removed